### ISSUE TITLE
library invalid

### ISSUE THREAD

**Issue by hadmut** [josephpal/esp32-Encrypt#4]
Hi, 
the Arduino IDE 1.8.13 can't use the library and complains it to be invalid, because no *.h files in the top folder. I had to as a symlink from top folder to src/Cipher.h. 

Reason probably as described under 
https://forum.arduino.cc/t/arduino-cant-find-a-h-file/567630/4

which says:

> If the library is in 1.0 format (doesn't have a library.properties file), the Arduino IDE will only check the root folder. If the library is in 1.5 format (has library.properties), the Arduino IDE will check in the src subfolder of the library folder.

So it seems as if putting the *.h in a src subfolder it requires a library.properties file.

regards

**Comment by homonto** [comment]
that is correct - once you add library.properties file you can use as "usual" but actually author should do it and zip it again ;-)

**Comment by josephpal** [comment]
> that is correct - once you add library.properties file you can use as "usual" but actually author should do it and zip it again ;-)

You are completly right. Right now, it is not ready to be used as an arduino library. You have to put the class files (.h, .cpp) n the same directory as your ino project.

Greetings
